content/pages/changelog/index.md

```diff
@@ -13,6 +13,19 @@ For historical changelog entries, please see our
 [2019](page://changelog/2019) and
 [2018 release notes](page://changelog/2018).
 
+## 2020-07-20 / Updates to Search Relevance & Performance
+
+In January, updates were released to Box's search [tokenizer](wiki-tokenizer),
+improving the relevancy of search results and performance. This change does not
+effect how metadata is tokenized.
+
+Notable changes to the API-initiated searches include:
+* Ability to use "[stop words](wiki-stop-words)" (e.g. to do list)
+* Improved Japanese search
```

Comment:
Can you remind me how Japanese search was improved? Like the tokenization is more accurate or the way a search is resolves has been improved?

Reply:
I'll get more details from Rebecca. The DDOC just had a link to the community post so I borrowed the language from that.